value_replacer/value_replacer.rb

```diff
@@ -0,0 +1,68 @@
+require 'csv'
+
+# Creates a hash where the key is the value you want to replace
+# and the value is the value with which to replace
+# PATH is a string to the folder path
+# FILE is a string of the filename
+# ENCODING (nil) is a string indicating the character encoding for the file
+# If you want the CSV library to detect the character encoding, do not provide encoding
+def create_value_hash (path,file,encoding=nil)
+  filepath = File.join(path,file)
+  value_hash = {}
+  
+  CSV.foreach(filepath,encoding: encoding) do |row|
+    value_hash[row[0]] = row[1]
+  end
+
+  value_hash
+end
+
+# Given a CSV and a hash where the key is the value you want to replace
+# and the value is the value with which to replace,
+# create a new .csv file and replace values
+# PATH (string) is the folder path
+# FILE (string) is the filename
+# VALUE_HASH (hash) is a hash where the key is the value you want to replace
+# and the value is the value with which to replace. This is meant to be created using
+# create_value_hash
+# INDEXES(int or array of ints) are the indexes of the values to replace
```

Comment:
I'm glad you pointed this out. I will try to be more clear about this. When I read in a CSV I use the array of arrays structure the CSV library provides. `INDEXES` refers to the indexes within each row (array).

In this case it's confusing because the example implementation is derived from an actual case and not the test data in input.csv. I'll update the implementation to match the test data.